### DISCUSSION TITLE
Issue initializing the API?

### DISCUSSION THREAD

**Discussion by tomasrivero** [wxyz-abcd/node-haxball#57] [General]
Running the node app throws the following error `Cannot read properties of undefined (reading 'create')`

![image](https://user-images.githubusercontent.com/31349354/222925572-ac540d68-d4ee-4b58-8caf-0bf62637a751.png)

I'm surely missing something, but I'm not sure what exactly, considering how early I am on the development process

**Comment by wxyz-abcd** [comment]
Node.js package is not updated yet, so its version is incompatible with the current documentation. I will probably update it next week. In the mean time, you can work inside the browser. Or, if you prefer, you can read the old documentation here, but it does not have some of the current features: https://www.npmjs.com/package/node-haxball
